packages/core/src/generators/angular.ts

```diff
@@ -335,7 +335,25 @@ export const componentToAngular: TranspilerGenerator<ToAngularOptions> =
           stateVars,
         }),
     });
-
+    // Preparing built in component metadata parameters
+    const componentMetadata: Record<string, any> = {
+      selector: `'${kebabCase(json.name || 'my-component')}'`,
+      template: `\`
+        ${indent(template, 8).replace(/`/g, '\\`').replace(/\$\{/g, '\\${')}
+        \``,
+      ...(css.length ? { styles: `\`${indent(css, 8)}\`` } : {}),
```

Comment:
You removed the `[]` brackets in this string, this breaks the rendering of the styles property. you can see that the snapshot tests are all shifted.

```suggestion
      ...(css.length ? { styles: `[\`${indent(css, 8)}\`]` } : {}),
```

Make sure to run `yarn snapupdate` in `packages/core` locally to update test snapshots and see if there are changes there